imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -129,6 +139,17 @@ de_time:
   UNITS: microseconds
   LABLAXIS: DE time
 
+esa_step:
+  <<: *de_default
+  <<: *default_uint8
+  VALIDMIN: 1
+  VALIDMAX: 7
+  FORMAT: I1
+  CATDESC: Energy Step
+  DEPEND_1: esa_step
+  FIELDNAM: Energy Step
+  LABLAXIS: ESA
```

Comment:
It looks like the only difference between `esa_step_coord` and `esa_step` is that the former has `LABL_PTR_1: esa_step_label`? You could `esa_step` the basis for `esa_step_coord` so that it looks like:
```
esa_step: &esa_step
  <<: *de_default
  <<: *default_uint8
  VALIDMIN: 1
  VALIDMAX: 7
  FORMAT: I1
  CATDESC: Energy Step
  DEPEND_1: esa_step
  FIELDNAM: Energy Step
  LABLAXIS: ESA

esa_step_coord:
  <<: *esa_step
  LABL_PTR_1: esa_step_label
```

Reply:
That's a good suggestion. The only issue is `esa_step` is using `de_default` while `esa_step_coord` is using `coord_default`. The `de_default` sets a `DEPEND_1` that the `esa_step_coord` isn't using